healthcare-management/src/app/app-config/app.config.ts

```diff
@@ -1,7 +1,7 @@
 
 import {AppConfig} from './app.config.interface';
 import {InjectionToken} from '@angular/core';
-import {environment} from '../../environments/environment';
+import {environment} from '../../environments/environment.development';
```

Comment:
aici trebuie lasat environment simplu, deoarece in productie se foloseste environment, am redenumit fisierul din environment.development in environment. Am uitat sa precizez asta